src/app/activitywidget.cpp

```diff
@@ -223,10 +223,30 @@ void ActivityWidget::updateControls()
         mUi->btStartStop->setText(tr("Start"));
         mUi->btStartStop->setIcon(QIcon(":/icons/play"));
         mUi->lbDurationTime->clear();
+
+        window()->setWindowIcon(QIcon(":/icons/kemai"));
+
+        // TODO: probably not the best way to do this, needs improvement
+        MainWindow* mW = static_cast<MainWindow*>(getMainWindow());
```

Comment:
A Qt signal like "currentActivityChanged(bool started)" or "currentActivityStarted()/Stoped()" will be cleaner. Then mainwindow can listen to this signal and change icon as needed.

Reply:
You're making me work over here... 😓
I'm not really a QT dev.